omod/src/main/java/org/openmrs/module/metadatasharing/web/controller/ImportController.java

```diff
@@ -283,6 +283,11 @@ public String uploadPOST(UploadForm uploadForm, Errors errors, Model model) thro
 				importer.loadSerializedPackageStream(in);
 				in.close();
 			}
+			catch(Exception e){
+				log.error(e);
+				errors.rejectValue("file","metadatasharing.error.package.file.invalid")
```

Comment:
Missing semicolon!

Make sure you run compilation and tests before opening a PR, especially since we don't have consistent CI builds to check our work.

Reply:
Ohh. I am so sorry for this. I am fairly certain that the After image has the compiled version of the code. So I am not sure how this error propped up. I guess this might be result of me typing ctrl z to test out minor code changes before commiting.

Reply:
I have added the semicolon. Sorry for the inconvenience